app/assets/javascripts/ama_layout/desktop/sticky-footer.coffee

```diff
@@ -1,6 +1,16 @@
 $(window).bind "load", ->
-  footer = $("footer")
-  position = footer.position()
-  footer_top = if position is undefined then 0 else position.top
-  height = $(window).height() - footer_top - footer.height()
-  footer.css({ 'margin-top': height + 'px' }) if height > 0
\ No newline at end of file
+  adjustFooter = ->
+    $("footer").css({ 'margin-top': calcHeight() + 'px' }) if calcHeight() > 0
```

Comment:
I don't think the `if calcHeight() > 0` condition will ever be false, since the function should return nothing less than 50.